presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -50,6 +50,8 @@
 import static java.lang.String.format;
 import static org.testng.Assert.assertEquals;
 
+// TODO: enable once https://github.com/prestosql/presto/issues/5120 is resolved
+@Test(enabled = false)
```

Comment:
nit: we have `@Flaky` annotation for this purpose

Reply:
We could also use that @raunaqmorarka, although I think it can fail differently depending if stats are not available at all or just portion of it is missing